### PR TITLE
split repository code

### PR DESCRIPTION
**Pull request by ryanfkeepers**
splits the code in repository/repository for cleanliness. No logic changes, just code movement.  This is prep for the nest PR.

---

#### Does this PR need a docs update or release note?

- [x] :no_entry: No

#### Type of change

- [x] :broom: Tech Debt/Cleanup

#### Issue(s)

* #2025

### CODE REVIEW

**Review by neha-Gupta1 — APPROVED** [review]
Just code movement.

File: src/pkg/repository/repository.go [hunk 1]
```diff
@@ -237,6 +184,11 @@ func (r *repository) Connect(ctx context.Context) (err error) {
 		}
 	}()
 
+	// ctrl, err = connectToM365(ctx, sel.PathService(), r.Account, r.Opts)
```
Comment (pandeyabs) [inline src/pkg/repository/repository.go:187]: Is this commented out on purpose?

**Review by pandeyabs — APPROVED** [review]
Looks good. Nice split work! Left a comment on some commented out code.

File: src/pkg/repository/repository.go (hunk 1 above)
Comment (ryanfkeepers) [reply src/pkg/repository/repository.go:187]: Oh!  Heh, yes.  Was supposed to be deleted.  But it'll get patched up in the following PR either way.  Good call out.

**Pull request merged**